clusterloader2/pkg/measurement/manager.go

```diff
@@ -20,6 +20,12 @@ import (
 	"sync"
 
 	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/perf-tests/clusterloader2/pkg/config"
+)
+
+var (
+	// ClusterConfig represents cluster configuration.
+	ClusterConfig *config.ClusterConfig
```

Comment:
I'm not a fan of this global variable.

Can you make add a TODO to make this a class variable or parameter somewhere or something like that?

Reply:
TODO added.

Reply:
Instead of TODO(krzysied) we should move towards TODO(issue id).
Please open an issue for that and replace.

Reply:
Done.
I thought that putting TODO creator login is a best practice is k8s repo.